lib/components/ms-progress-bar/MsProgressBar.vue

```diff
@@ -0,0 +1,49 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="progress-bar-container">
+    <div class="progress-bar">
+      <div class="completed" />
+    </div>
+    <ion-text class="title-h5 progress-text">{{ progression }}</ion-text>
+  </div>
+</template>
+
+<script setup lang="ts">
+defineProps<{
+  progression?: string;
```

Comment:
Given their names and the values that are used as an example in HomePage, these two should be numbers, not strings.

Reply:
@Max-7 here we pass a string: `'79%'`.
When I tried to pass only '79' and add the '%' directly in the SCSS, it wasn't working